packages/appstore/src/assets/svgs/currency/index.tsx

```diff
@@ -3,30 +3,36 @@ import { Icon } from '@deriv/components';
 import { capitalizeFirstLetter } from '@deriv/shared';
 import { IconProps } from '../icon-types';
 
+export const IconCurrency = (icon?: string): icon is Currency =>
+    Currencies.find(currency => currency === icon) !== undefined;
+
+const Currencies = [
+    'AUD',
+    'BCH',
+    'BTC',
+    'BUSD',
+    'DAI',
+    'ETH',
+    'EURCHECK',
+    'EUR',
+    'EURS',
+    'EUSDT',
+    'GBP',
+    'IDK',
+    'LTC',
+    'PAX',
+    'TUSD',
+    'TUSDT',
+    'UNKNOWN',
+    'USD',
+    'USDC',
+    'USDK',
+    'UST',
+    'VIRTUAL',
+] as const;
```

Comment:
Since i changed the useStores to `useStore`, was getting TypeError in the icon component. The `CurrencySwitcherContainer` icon prop could be undefined. Therefore this solution.